ldms/src/sampler/aries_mmr/aries_lbw.c

```diff
@@ -0,0 +1,598 @@
+/* -*- c-basic-offset: 8 -*-
+ * Copyright (c) 2015-2017 National Technology & Engineering Solutions
```

Comment:
Shouldn't the year be 2020?

Reply:
I'm seeing a lot of samplers that are 2017/2018 in OVIS-4, perhaps this needs a refresh for all?